src/index.ts

```diff
@@ -568,7 +609,9 @@ class Update<
         dynamicSetClauses.push(`${col} = $${n}`);
       }
       const setSql = dynamicSetClauses.join(', ');
-      const query = `UPDATE ${this.table} SET ${setSql}${whereClause}${limitClause} RETURNING *`;
+      let query = `UPDATE ${this.table} SET ${setSql}${whereClause}${limitClause} RETURNING *`;
+      query = updateQueryWithIsNull(query, whereVals, whereNames);
+      console.log(query);
```

Comment:
Did you mean to leave this here?

Reply:
Whoops! Thanks for the catch @kamaldon. Fixed in a84b45234a0535d17354f36ebc0a5a695691e564, released in 0.1.3 and #24 should keep it from happening again.